server/src/main/java/org/opensearch/identity/MyShiroModule.java

```diff
@@ -0,0 +1,118 @@
+package org.opensearch.identity;
+
+import org.apache.shiro.authc.AuthenticationException;
+import org.apache.shiro.authc.UsernamePasswordToken;
+import org.apache.shiro.authz.Permission;
+import org.apache.shiro.authz.permission.AllPermission;
+import org.apache.shiro.authz.permission.RolePermissionResolver;
+import org.apache.shiro.authz.permission.WildcardPermission;
+import org.apache.shiro.mgt.SecurityManager;
+import org.apache.shiro.SecurityUtils;
+import org.apache.shiro.realm.Realm;
+import org.apache.shiro.realm.SimpleAccountRealm;
+import org.apache.shiro.mgt.DefaultSecurityManager;
+import org.apache.shiro.subject.SimplePrincipalCollection;
+import org.apache.shiro.subject.Subject;
+import org.opensearch.common.Strings;
+
+import java.util.Base64;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Optional;
+
+/**
+ * Prototyping use of identity management (authentication & authorization) implemented with Shiro
```

Comment:
Having a single `&` here was causing issues running `./gradlew clean assemble` on my local machine. I had to switch it to `&amp;` to get the command to run.

Reply:
There's always "and" :-)

Also might want to use the common IAM terminolity "identity and access management".